src/randomCardsIds.js

```diff
@@ -0,0 +1,18 @@
+import { random, shuffle } from "lodash";
+
+const AMOUNT_OF_PHOTOS = 992;
+
+const randomCardsIds = length => {
```

Comment:
Why is this whole thing a separate file? Is this used in many places?

Reply:
No, it insn't. I'll start with putting in the BoardView component file, and tray to refactor it in more functional way.

Reply:
:+1: